orbit/cmd/orbit/orbit.go

```diff
@@ -1186,6 +1196,28 @@ func main() {
 				c.String("fleet-desktop-alternative-browser-host"),
 				opt.RootDirectory,
 			)
+			go func() {
+				for {
+					msg := <-desktopRunner.errorNotifyCh
```

Comment:
Could you also write these errors to the logger so they appear in the `fleetd_logs` table? 🙂

Reply:
Done

Reply:
Put the error in a `.error(errors.New(msg)).Msg("fleet-desktop error")` or something and we're good to go